modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -861,11 +867,16 @@ impl Wallet {
         self.btc_rpc.get_fee_rate(CONFIRMATION_TARGET).await
     }
 
-    pub async fn get_fee_rate(&self) -> anyhow::Result<Feerate> {
-        Ok(self
-            .get_fee_rate_opt()
-            .await?
-            .unwrap_or(self.cfg.consensus.default_fee))
+    /// Returns the fee rate from Bitcoind or the default if the node is
+    /// unreachable or isn't able to provide an estimate
+    pub async fn get_fee_rate(&self) -> Feerate {
```

Comment:
IMO, we should return `Option` here, and on error a `None`, which would make consensus code just not change the vote at all (use previous one). Otherwise we might get into a lot of noise and choppy voting - with fee rate bouncing on every intermittent error etc.

Reply:
Good call. I removed the existing function since it's only used in `consensus_proposal` and the abstraction was a bit awkward (`get_fee_rate_or_default_or_none_on_error`?). This seems much easier to read.